### PR TITLE
Add follow settings

### LINKED ISSUE #56: Follow toggle

### PR DESCRIPTION
**Pull request by nwingt**
Resolve #56

### CODE REVIEW

File: app/models/reader-store/reader-store.ts [hunk 1]
```diff
@@ -115,6 +122,33 @@ export const ReaderStoreModel = types
...
+      self.isFetchingCreatorList = true
+      try {
+        const result: ReaderCreatorsResult = yield self.env.likerLandAPI.fetchReaderCreators()
+        switch (result.kind) {
+          case "ok":
+            self.followingCreators.replace([])
+            result.following.forEach(likerID => {
+              const creator = self.createCreatorFromLikerId(likerID)
+              creator.isFollowing = true
+              self.followingCreators.push(creator)
```
Comment (williamchong007) [inline app/models/reader-store/reader-store.ts]: Would pushing these creators into a local scope array, then call `self.followingCreators.replace()` better then replacing the prop then push it one by one?

File: app/screens/reader-screen/reader-screen.tsx [hunk 2]
```diff
@@ -44,6 +42,62 @@ export class ReaderScreen extends React.Component<ReaderScreenProps> {
...
+
+          // Notify follow status to user
+          const { isFollowing: isFollowingNow } = content.creator
+          if (
+            // Follow status changes
+            isFollowing !== isFollowingNow &&
+            // Not include unfollow in following list
+            !(
+              !isFollowingNow &&
+              this.props.navigation.state.routeName === "Following"
```
Comment (williamchong007) [inline app/screens/reader-screen/reader-screen.tsx]: Do the `Following` screen use this action? if not why this guard?

File: app/screens/reader-screen/reader-screen.tsx [hunk 3]
```diff
@@ -44,6 +42,62 @@ export class ReaderScreen extends React.Component<ReaderScreenProps> {
...
+            )
+          ) {
+            Alert.alert(
```
Comment (williamchong007) [inline app/screens/reader-screen/reader-screen.tsx]: Not sure if we need to alert user everytime

File: app/models/reader-store/reader-store.ts (hunk 1 above)
Comment (nnkken) [reply app/models/reader-store/reader-store.ts]: https://mobx.js.org/refguide/action.html
> actions will batch mutations and only notify computed values and reactions after the (outer most) action has finished

File: app/components/content-list-item/content-list-item.props.ts [hunk 4]
```diff
@@ -37,4 +38,14 @@ export interface ContentListItemProps {
    * A callback when the undo button is pressed.
    */
   onPressUndoButton?: (content: Content) => void
+
+  /**
+   * A callback when the undo button is pressed.
```
Comment (williamchong007) [inline app/components/content-list-item/content-list-item.props.ts]: comment not updated

File: app/components/content-list-item/content-list-item.props.ts [hunk 5]
```diff
@@ -37,4 +38,14 @@ export interface ContentListItemProps {
...
+   */
+  onSwipeOpen?: (key: string, ref: React.RefObject<SwipeRow<{}>>) => void
+
+  /**
+   * A callback when the undo button is pressed.
```
Comment (williamchong007) [inline app/components/content-list-item/content-list-item.props.ts]: comment not updated

**Pull request merged**